crud/users.py

```diff
@@ -202,3 +203,73 @@ async def set_new_role(uid, new_role, user_role, db: AsyncSession):
     await db.commit()
     return {'status': 'success',
             'message': f'User {uid} successfully changed to {new_role.role}.'}
+
+
+@handle_errors
+async def set_reminder(user_id, reminder_set, db: AsyncSession):
+    """
+    Function to set the reminders on or off for the user.
+
+    Returns:
+        Message of the transaction.
+    """
+    query = sa.select(User).where(User.id == user_id)
+    result = await db.execute(query)
+    user = result.scalar()
+    if user.daily_reminder == reminder_set and reminder_set == "yes":
+        raise HTTPException(status_code=200,
+                            detail='Reminders are already enabled for this user')
```

Comment:
raise HttpException with 200 status code seems a bit contradictory.
Maybe we could either return  normally with message: "Remainders already enabled/disabled for this user" or we could raise 400 (bad request) with same detail